scripts/sanitycheck

```diff
@@ -3317,6 +3322,16 @@ def main():
         error("west-flash requires device-testing to be enabled")
         sys.exit(1)
 
+    if os.name == 'nt' and \
+       (not options.build_only or not options.disable_size_report):
+        print("Testing with sanitycheck on Windows is not fully supported yet.")
+        print("Only build without size report works for now:")
+        print("\t- '--build-only --disable-size-report'")
+        print("\t- Requires make: 'choco install make'")
+        print("\t- May require long-path support in Windows OS.")
```

Comment:
Similar comment: I don't see why this  information is tied to --build-only and --disable-size-report (and missing from issue #2664?)